### ISSUE TITLE
More methods used in examples?

### ISSUE THREAD

**Issue by mithru** [processing/p5.accessibility#54]
#### Nature of issue?

- [ ] Existing feature enhancement

#### Feature enhancement details:
Examples can show arcs, lines, text, etc. Makes it easier for anyone cloning this repo to test as well.

**Comment by MathuraMG** [comment]
That sounds good, I just want to make sure that a first time user of p5.accessibility doesn't get overwhelmed though? @lm-n

**Comment by mithru** [comment]
Makes sense. What about leaving the current example alone and making a bunch of mini examples for each method? Or having just one big example that holds everything (in addition to the existing simple one)? 

I'm currently thinking towards the former i.e. a bunch of mini examples, because when we eventually implement ways to navigate items in massive for loops, and OOP stuff, that might need its own example anyway? 

What are both your thoughts?

**Comment by lm-n** [comment]
I think that would be ideal. To have straight forward example for beginners that only deals with the main functionality of the method and then the more complex ones.
